lib/ralyxa/request_entities/request.rb

```diff
@@ -15,7 +15,7 @@ class Request
 
       def initialize(original_request, user_class = Ralyxa::RequestEntities::User)
         @request = JSON.parse(original_request.body.read)
-        original_request.body.rewind
+        original_request.body&.rewind
```

Comment:
This is a good solution but the intent isn't so clear. Can we use a private method like `conditionally_rewind_request_if_rack_didnt_do_it(request)`?

Reply:
I'll update as per your comments